pkg/cmd/server/apis/config/v1/conversions.go

```diff
@@ -136,6 +136,13 @@ func SetDefaults_MasterConfig(obj *MasterConfig) {
 			obj.AdmissionConfig.PluginConfig[pluginName] = &AdmissionPluginConfig{}
 		}
 	}
+
+	// Set defaults Cache TTLs for external Webhook Token Reviewers
+	for i := range obj.AuthConfig.WebhookTokenAuthenticators {
+		if len(obj.AuthConfig.WebhookTokenAuthenticators[i].CacheTTL) == 0 {
+			obj.AuthConfig.WebhookTokenAuthenticators[i].CacheTTL = "2m"
```

Comment:
The node defaults to 5m.  Any reason for a different default?  If this was already litigated, I don't feel strongly.

Reply:
upstream defauls to 2m for CacheTTL see other comments made previously

Reply:
> upstream defauls to 2m for CacheTTL see other comments made previously

Ok.